### PR TITLE
[feature](docker)add docker-iceberg init tables

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

Add some init tables for docker-iceberg.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wuwenchi** [comment]
run buildall

File: docker/thirdparties/docker-compose/iceberg/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+```
+data:
+    input: 初始化docker使用的数据
```
Comment (morningman) [inline docker/thirdparties/docker-compose/iceberg/README.md:23]: use English

File: docker/thirdparties/docker-compose/iceberg/iceberg.yaml.tpl [hunk 2]
```diff
@@ -18,19 +18,18 @@
 version: "3"
 
 services:
-  doris--spark-iceberg:
+  spark-iceberg:
```
Comment (morningman) [inline docker/thirdparties/docker-compose/iceberg/iceberg.yaml.tpl:21]: Why remove the prefix?

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Review by zhangguoqiang666 — APPROVED** [review]
LGTM

**Review by wsjz — APPROVED** [review]
LGTM

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by wuwenchi** [comment]
run build all

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Comment by zhangguoqiang666** [comment]
run external

**Comment by zhangguoqiang666** [comment]
run arm

**Review by morningman — APPROVED** [review]

**Pull request merged**